colors/spaceduck.vim

```diff
@@ -82,9 +144,33 @@ if &background == 'dark'
   call s:hi('SpaceduckCyan',        s:palette.cyan,        s:palette.none)
   call s:hi('SpaceduckMagenta',     s:palette.magenta,     s:palette.none)
 
+  call s:h('SpaceduckRedBold',         s:palette.red,         s:palette.none, [s:attrs.bold])
+  call s:h('SpaceduckOrangeBold',      s:palette.orange,      s:palette.none, [s:attrs.bold])
+  call s:h('SpaceduckGreenBold',       s:palette.green,       s:palette.none, [s:attrs.bold])
+  call s:h('SpaceduckYellowBold',      s:palette.yellow,      s:palette.none, [s:attrs.bold])
+  call s:h('SpaceduckPurpleBold',      s:palette.lavender,    s:palette.none, [s:attrs.bold])
+  call s:h('SpaceduckDarkPurpleBold',  s:palette.darkpurple,  s:palette.none, [s:attrs.bold])
+  call s:h('SpaceduckPurple2Bold',     s:palette.purple2,     s:palette.none, [s:attrs.bold])
+  call s:h('SpaceduckDarkPurple2Bold', s:palette.darkpurple2, s:palette.none, [s:attrs.bold])
+  call s:h('SpaceduckCyanBold',        s:palette.cyan,        s:palette.none, [s:attrs.bold])
+  call s:h('SpaceduckMagentaBold',     s:palette.magenta,     s:palette.none, [s:attrs.bold])
+  
+  call s:h('SpaceduckRedItalic',         s:palette.red,         s:palette.none, [s:attrs.italic])
+  call s:h('SpaceduckOrangeItalic',      s:palette.orange,      s:palette.none, [s:attrs.italic])
+  call s:h('SpaceduckGreenItalic',       s:palette.green,       s:palette.none, [s:attrs.italic])
+  call s:h('SpaceduckYellowItalic',      s:palette.yellow,      s:palette.none, [s:attrs.italic])
+  call s:h('SpaceduckPurpleItalic',      s:palette.lavender,    s:palette.none, [s:attrs.italic])
+  call s:h('SpaceduckDarkPurpleItalic',  s:palette.darkpurple,  s:palette.none, [s:attrs.italic])
+  call s:h('SpaceduckPurple2Italic',     s:palette.purple2,     s:palette.none, [s:attrs.italic])
+  call s:h('SpaceduckDarkPurple2Italic', s:palette.darkpurple2, s:palette.none, [s:attrs.italic])
+  call s:h('SpaceduckCyanItalic',        s:palette.cyan,        s:palette.none, [s:attrs.italic])
+  call s:h('SpaceduckMagentaItalic',     s:palette.magenta,     s:palette.none, [s:attrs.italic])
+
```

Comment:
Looks good, ignore my previous comment about "if statements" and whatever on Discord.